.bcr/presubmit.yml

```diff
@@ -0,0 +1,9 @@
+tasks:
+  verify_targets:
+    name: Verify targets
+    platform: macos
+    build_targets:
+      - '@yams//:Yams'
+      - '@yams//Tests/...'
```

Comment:
can remove this since testing will build everything as long as there aren't build only targets in there, nbd either way